src/email/components/email_body.R

```diff
@@ -14,7 +14,6 @@ box::use(./line_block)
 #' @param title Shock title text to go next to HDX Signals at the top of the email
 #' @param iso3 Vector of ISO3 codes
 #' @param country Vector of country names
-#' @param alert_level Vector of alert levels
```

Comment:
We're removing this for all indicators now, right?

Reply:
Yes! I promise these things will be tracked in the long run haha.